cmd/zpool/zpool_main.c

```diff
@@ -2357,6 +2358,35 @@ health_str_to_color(const char *health)
 	return (NULL);
 }
 
+/*
+ * Called for each leaf vdev.  Returns 0 if the vdev is healthy.
+ * A vdev is unhealthy if any of the following are true:
+ * 1) there are read, write, or checksum errors,
+ * 2) its state is not ONLINE, or
+ * 3) slow IO reporting was requested (-s) and there are slow IOs.
+ */
+static int
+vdev_is_healthy(void *hdl_data, nvlist_t *nv, void *data)
```

Comment:
Typically functions that have 'is' in their names return true if the 'is' thing is true.  That way you can use it directly in an `if` statement:
```
if (vdev_is_healthy())
    printf("the vdev is healthy")
```

So I'd reccomend could either name this `vdev_is_unhealthy()` or something like `vdev_health_check_cb()` since it's a callback function.

Reply:
Good catch.  Let's go with `vdev_health_check_cb()` to make it clear this is a callback function.